accounts/keystore/keystore.go

```diff
@@ -286,7 +286,7 @@ func (ks *KeyStore) SignTx(a accounts.Account, tx *types.Transaction, chainID *b
 	// Depending on the presence of the chain ID, sign with EIP155 or homestead
 	if chainID != nil {
 		//return types.SignTx(tx, types.NewEIP155Signer(chainID), unlockedKey.PrivateKey)
-		return types.SignTx(tx, types.NewEIP155Signer(big.NewInt(777)), unlockedKey.PrivateKey)
+		return types.SignTx(tx, types.NewEIP155Signer(big.NewInt(69420)), unlockedKey.PrivateKey)
```

Comment:
I'll vote for a chain ID of '787'

Not too far from 777, and it's a palindromic prime!